content/quickstart.md

```diff
@@ -7,29 +8,72 @@ tocopen: true
 
 # Prerequisites
 
-Under construction.
+1- OpenSeeFace tracking: webcam
 
-# Installation
+2- IFacialMocap tracking: iOS device with face ID or A12 (or newer) chip
+
+3- MeowFace tracking: android device that supports ARCore (most modern phones do)
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+4- 64-bit Operating system with x86 architecture
+
+# Installation
 
 ## Windows
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+1- download windows [zip file](https://github.com/virtual-puppet-project/vpuppr/releases)

```

Comment:
Sentences should be capitalized. However in a list, sentences do not necessarily need to end in a period